inference-engine/tests_deprecated/functional/shared_tests/CMakeLists.txt

```diff
@@ -13,25 +13,11 @@ list(APPEND SHARED_LIBRARIES
         ngraphFunctions
         )
 
-if(USE_CNNNETWORK_LPT)
-    file(GLOB SHARED_TESTS_SRC
+file(GLOB SHARED_TESTS_SRC
         ${CMAKE_CURRENT_SOURCE_DIR}/common_single_layer_tests/*.cpp
         ${CMAKE_CURRENT_SOURCE_DIR}/lstm/*.cpp
         ${CMAKE_CURRENT_SOURCE_DIR}/graph_tools/*.cpp
-        # requires legacy LPT
-        ${CMAKE_CURRENT_SOURCE_DIR}/network_tests/*.cpp
-        ${CMAKE_CURRENT_SOURCE_DIR}/transformations/*.cpp
-        ${CMAKE_CURRENT_SOURCE_DIR}/transformations/*.hpp
-        ${CMAKE_CURRENT_SOURCE_DIR}/transformations/common/*.cpp
```

Comment:
what about these files? 
```
        ${CMAKE_CURRENT_SOURCE_DIR}/network_tests/*.cpp
        ${CMAKE_CURRENT_SOURCE_DIR}/transformations/*.cpp
        ${CMAKE_CURRENT_SOURCE_DIR}/transformations/*.hpp
        ${CMAKE_CURRENT_SOURCE_DIR}/transformations/common/*.cpp
```
And the same for MKLDNN plugin tests

Reply:
done, thanks